team-norms/supported_software.md

```diff
@@ -0,0 +1,25 @@
+# Supported Software
+
+Where the day-to-day running of an application needs to transition to our support team, there are specific considerations you must take when making technology choices.
+
+As an organisation we're keen to embrace new and diverse technology where there's a strong commercial justification.
+
+When you're considering a major technology choice that isn't documented here, before starting a production implementation you must seek agreement from the support team that they're able to provide BAU support.
```

Comment:
I personally think this is going to discourage teams from breaking from the list. While I agree that our software has to be supportable, I can see this being a barrier to entry to "diverse technology".

Reply:
I agree. I think there's a conflict between standardisation from a support perspective and desire to be more technology diverse. Suggestions for other ways to handle this conflict?